packages/lodestar/src/logger/winston.ts

```diff
@@ -28,9 +29,27 @@ export class WinstonLogger implements ILogger {
             format.timestamp({
               format: "YYYY-MM-DD HH:mm:ss"
             }),
-            format.printf(
-              info => `${info.timestamp} [${info.module.toUpperCase()}] ${info.level}: ${info.message}`
-            )
+            format.printf((info) => {
+              // const screenSize = process.stdout.columns;
+              // const paddingBeforeInfo = 21;
+              const paddingBetweenInfo = 30;
+              // const paddingPreMsg = paddingBeforeInfo + paddingBetweenInfo + info.level.length - 19;
+
+              const infoString = (info.module || info.namespace || "");
+              const infoPad = paddingBetweenInfo - infoString.length;
+
+              // if (info.message.length + paddingPreMsg > screenSize) {
+              //   const max = screenSize - paddingPreMsg;
+              //   const p2 = info.message.length - max;
+              //   const msgs = [info.message.substring(0, max), info.message.substring(max)];
+              // eslint-disable-next-line max-len
+              //   return (`${info.timestamp}  [${infoString.toUpperCase()}] ${info.level.padStart(infoPad)}: ${msgs[0]}\n${msgs[1].padStart(paddingPreMsg + 2)}`)
+              // } else {
```

Comment:
what are these comments for? still needed?

Reply:
I thinks thats @GregTheGreek log padding?

Reply:
Hahaha , yeah that's mine. We can remove it.